core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java

```diff
@@ -28,9 +29,15 @@
 
     public static final String FEATURE_HTTP_CLIENT = "dataspaceconnector:http-client";
     public static final String FEATURE_RETRY_POLICY = "dataspaceconnector:retry-policy";
-    private static final String MAX_RETRIES = "dataspaceconnector:core.retry.max-retries";
-    private static final String BACKOFF_MIN_MILLIS = "dataspaceconnector.core.retry.backoff.min";
-    private static final String BACKOFF_MAX_MILLIS = "dataspaceconnector.core.retry.backoff.max";
+
+    @EdcSetting
+    private static final String MAX_RETRIES = "edc.core.retry.max-retries";
```

Comment:
Hi Benjamin, do you think we should also change max-retries to max.retries for consistency? I don't have any particular preference, I just wanted to raise it as a qyestion

Reply:
Hi @jimmarino 
if we want to be consistent with the others settings just below (e.g. edc.core.retry.backoff.min...) it should even be edc.core.retry.retries.max :)
I dont have preference either, let me know which version you prefer and I'll made the change.

Reply:
That seems reasonable

Reply:
@jimmarino renamed as discussed

Reply:
@MoritzKeppler are you OK with this? If so, I'll merge